src/Razor/src/Microsoft.VisualStudio.Editor.Razor/BackgroundParser.cs

```diff
@@ -30,24 +28,15 @@ public BackgroundParser(RazorProjectEngine projectEngine, string filePath, strin
         /// <summary>
         /// Fired on the main thread.
         /// </summary>
-        public event EventHandler<BackgroundParserResultsReadyEventArgs> ResultsReady;
+        public event EventHandler<BackgroundParserResultsReadyEventArgs>? ResultsReady;
 
-        public bool IsIdle
-        {
-            get { return _main.IsIdle; }
-        }
+        public bool IsIdle => _main.IsIdle;
 
-        public void Start()
-        {
-            _bg.Start();
-        }
+        public void Start() => _bg.Start();
 
-        public void Cancel()
-        {
-            _main.Cancel();
-        }
+        public void Cancel() => _main.Cancel();
```

Comment:
I appreciate the effort of the nullable PRs, but I don't like the style changes that come along for the ride. The changes from `!= null` to `is not null` are good, the changes from a null check to `?.` i'm okay with, and the changes from a null check to `??=` I guess I can look past, though I don't prefer, but changing methods or properties to expression bodied is a step too far for me, personally.

Can you revert these, and try to avoid them in future?

And to be clear, I'm not going to comment on every change to an expression bodied method, but you should assume this comment applies to them all.

Reply:
I knew that such changes aren't null-related, but my main argument was "This code would probably be never touched again and style-only PRs are not preferable, so maybe I can remove such legacy-looking code (personally for me, espacially properties with `get` and `return`) in a bulk as a side result". However as you don't like this, I am totally ok to revert them

Reply:
I might be old fashioned, but I've never heard anyone describe methods with bodies as "legacy" code 😛😛